### ISSUE TITLE
Vagrant 2.2.7 does int install on Windows

### ISSUE THREAD

**Issue by GhaziTriki** [hashicorp/vagrant#11382]
### Vagrant version
2.2.7

### Host operating system
Windows 10

### Guest operating system
Nothing

### Expected behavior
The "vagrant_2.2.7_x86_64.msi" should install.

### Actual behavior
Double clicking on the executable does not start it. I tried with "vagrant_2.2.6_x86_64.msi" and it works.

**Comment by iiRobin** [comment]
I have kind of the same problem. I, on the other hand, can execute "vagrant_2.2.7_x86_64.msi". When it's done installing and I restart my PC and I type 'vagrant version' it doesn't say the 2.2.7 is installed.

**Comment by chrisroberts** [comment]
@GhaziTriki When double clicking on the msi installer file, does Windows provide any feedback? If Windows is not providing a clear message on why the installer is not being opened, it may be that virus scanning software is preventing it from being opened.

@iiRobin As you are experiencing a different issue than described here, please open a new issue providing the requested information (including a gist of the debug output from running `vagrant version --debug`). Thanks!

**Comment by h-haaks** [comment]
I'm having kind of the same issue installing vagrant_2.2.7_x86_64.msi  on windows 10.
I get this mesage ( translated from Norwegian ) "Windows Defender SmartScreen stopped an unknown app from starting....."

I think the problem could be related to the sha1 signature of the msi.
Is it possible for you to add an sha256 signature?

**Comment by pedrolamarao** [comment]
I have the same problem.

**Comment by doublenb** [comment]
Same here. Running installer for 2.2.7 brings up "Windows protected your PC" prompt stating:

"Windows Defender SmartScreen prevented an unrecognized app from starting."

I don't have the option of installing anyway (prevented by domain policies), so I can't install it. I am running 2.2.6 at the moment, which installed fine.

**Comment by gdemarcsek** [comment]
Any news on this? I've experienced the same problem.

**Comment by MaxvdBerk** [comment]
I encounter the same issues

**Comment by matthew-macgregor** [comment]
I'm also seeing the Microsoft Defender SmartScreen warning when trying to install Vagrant `2.2.7_x86_64`. 

![image](https://user-images.githubusercontent.com/1237132/79626912-b891db00-80f9-11ea-8679-0ae29584ace2.png)

It seems like this must be a false positive and needs to be evaluated for whitelisting with SmartScreen, presumably?

Checksum of the installer:
`SHA256: 6425E450EA180C00FBAB17BCA3E9CBD384072617A7ACABFF57C89B9EBC57FEB5`

Matches the checksum published for this build: https://releases.hashicorp.com/vagrant/2.2.7/vagrant_2.2.7_SHA256SUMS

**Comment by iamfraggle** [comment]
I'm in a position to try 'Run Anyway' but the button may as well be 'Fail Anyway'! :-D

After choosing a path to install (I'm using the defaults), it says click install to begin the installation.  When I click install I am presented with the error "Error reading from file <path_to_installer>\vagrant_2.2.7_x86_64.msi.  Verify that the file exists and that you can access it."

I have verified the hash of the installer.  It didn't give me a UAC prompt before the error, but I don't know if that's because it's failed before reaching the point that it would normally ask for it.

**Comment by chrisroberts** [comment]
Hi everyone. The SmartScreen warnings that are being generated are due to the signing certificate used when building and packaging Vagrant being updated just before the 2.2.7 release. Aside from checking the validity of the signature, SmartScreen uses a reputation score to determine "likely safe" files based on some unknown number of downloads and installs. The reputation score can apparently no longer be transferred from one certificate to another, so this warning is produced until some threshold has been reached.

**Comment by doublenb** [comment]
Found a **very** hacky solution. I double checked the checksum, then on the msi file, right click -> Properties -> General, right at the bottom theres a checkbox that says:

"This file came from another computer and might be blocked to help protect this computer"

Tick the "Unblock" checkbox then click Apply. It allows you to run the installer then.

**Comment by soapy1** [comment]
Hi there,

It looks like this has been resolved within a more recent shipped version of Vagrant so I am now closing this issue. If the original issue was not fully resolved, please reopen this issue or create a new one.

Cheers!
